src/app/wallets/get-balance-for-wallet.ts

```diff
@@ -30,7 +30,7 @@ export const getBalanceForWallet = async ({
 export const getBalanceForWalletId = async (
   walletId: WalletId,
 ): Promise<Satoshis | ApplicationError> => {
-  const balance = await LedgerService().getAccountBalance(walletId)
+  const balance = await LedgerService().getWalletBalance(walletId)
```

Comment:
As far as we discussed we agreed to go with `LedgerAccount` as the naming - so that would imply this function should be `getLedgerAccountBalanceForWalletId` or something like that.